src/redturtle/prenotazioni/behaviors/booking_folder/notifications/appio/__init__.py

```diff
@@ -95,6 +95,26 @@ def notify_on_refuse_appio_message_default_factory(context):
     )
 
 
+@provider(IContextAwareDefaultFactory)
+def notify_on_cancel_appio_subject_default_factory(context):
+    return api.portal.translate(
+        _(
+            "notify_on_cancel_appio_subject_default_value",
+            "[${prenotazioni_folder_title}] Booking canceled for ${title}",
+        )
+    )
+
+
+@provider(IContextAwareDefaultFactory)
+def notify_on_cancel_appio_message_default_factory(context):
+    return api.portal.translate(
+        _(
+            "notify_on_cancel_appio_message_default_value",
+            "The booking ${booking_type} of ${booking_date} at ${booking_time} was canceled.",
```

Comment:
Are you sure that these translations works? Because you are using a variable without its mapping

Reply:
Also in other places

Reply:
very sure

Reply:
is being substituted in the adapter, so even the translated message has those variables